### PR TITLE
Update integrated dx for 3 samples based on pathology review

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?

- update integrated_diagnosis column values for samples where initial molecular_subtype was "HGG, H3 wildtype" which got updated to molecular_subtype== "DMG K28M" from clinical review in 02-incorporate-clinical-feedback.Rmd

- update 03-incorporate-pathology-feedback.Rmd to add Notes to each pathology review updated sample_ids.

#### What was your approach?
- I am updating integrated_diagnosis column values for samples where initial molecular_subtype was "HGG, H3 wildtype"  OR " HGG, To be classified" which got updated to molecular_subtype== "DMG K28M" from clinical review 

- I'm also adding "Notes" = "Updated via OpenPBTA subtyping" in 03-incorporate-pathology-feedback.Rmd to cover all columns that need to be updated per updated molecular_subtype from pathology review. And specifically, since in v18 samples with "XYZ, To Be classified" the Notes and integrated_diagnosis is left NA (unlike in v17 histology file) so will create duplicate rows in v18 for same bs id (7316-506, 7316-765 and 7316-238) has Notes =c("NA", "Updated via OpenPBTA subtyping")  

#### What GitHub issue does your pull request address?
#878 

#### Which areas should receive a particularly close look?
Is there some condition I'm missing in clinical review update in 02?

#### Is there anything that you want to discuss further?
The LGAT update to "Recurrence" was not expected, it happened because I had v18 files in data release. I've updated to v17 files in the latest commit 3fd999b

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
compiled subtypes with updated int_dx for PNOC003 samples 

#### What is your summary of the results?
Update compiled subtypes

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by kgaonkar6** [comment]
hmm not sure why CI is erroring out with the EPN subtyping error that we see in v18 CI runs #876. In the "Checkout code" section of CI it does seem like we are using v17 histology file so not sure why it fails.

Regarding the actual error that fails CI : I believe (might be wrong) the error occurs because change in pathology_diagnosis from v17 to v18 discussed here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/876#issuecomment-749723223

**Comment by jharenza** [comment]
> hmm not sure why CI is erroring out with the EPN subtyping error that we see in v18 CI runs #876. In the "Checkout code" section of CI it does seem like we are using v17 histology file, oh maybe OPENPBTA_RELEASE=testing now has v18 testing files?
> 
> Regarding the actual error that fails CI : I believe (might be wrong) the error occurs because change in pathology_diagnosis from v17 to v18 discussed here: [#876 (comment)](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/876#issuecomment-749723223)

This looks good to me, but yes, we are now using v18 CI files, which I forgot about. I wonder if you should add then add this as an update within the v18 set of PRs?

**Review by jharenza — APPROVED** [review]
content looks right

**Comment by jharenza** [comment]
Hi @jaclyn-taroni @jashapiro @cansavvy - When looking at the final v18 histologies file, I found the error prompting this PR. We did as a standalone PR instead of within the v18 release PRs since it was a bug in the code. However, it errors out with V18 CI files because of the EPN subset step that @jashapiro has subsequently [fixed](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/876/commits/8f33c4a20eebaf89402003b91440b68f521fa4aa), but we do not yet have merged. 

>hmm not sure why CI is erroring out with the EPN subtyping error that we see in v18 CI runs #876. In the "Checkout code" section of CI it does seem like we are using v17 histology file, oh maybe OPENPBTA_RELEASE=testing now has v18 testing files?

The easiest thing may be for us to switch back to v17 CI files to get this run through CI, merge this, switch to v18 CI files (unless you are OK with us merging this without it passing V18 CI, so that we can get this updated into the V18 histologies files. Please advise :).

File: analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd [hunk 1]
```diff
@@ -316,6 +323,10 @@ compiled_df <- compiled_df %>%
     molecular_subtype = case_when(
       sample_id == "7316-1102" ~ "DMG, H3 K28",
       TRUE ~ molecular_subtype
+    ),
+    Notes = case_when(
+      sample_id == "7316-1102" ~ "Updated via OpenPBTA subtyping",
```
Comment (cansavvy) [inline analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd:328]: I don't quite understand why it looks like there's recoding by individual samples -- it would probably be good to look into a more systematic way to do this, but that should be in a different PR, not now.

**Review by cansavvy — APPROVED** [review]
Seems reasonable to me and like it is doing what is described.

File: analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd (hunk 1 above)
Comment (jaclyn-taroni) [reply analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd:328]: This notebook is designed to handle individual cases based on feedback from pathologists / clinical expertise. I think it's possible that coding for individual cases with lots of context is more clear.
